circle.yml

```diff
@@ -16,7 +16,6 @@ dependencies:
     - sudo service apache2 restart
     - chmod a+rwx backup log public/rss upload
     - sudo composer self-update --no-interaction
-    - composer config --global --no-interaction github-oauth.github.com $GITHUB_OAUTH_TOKEN
```

Comment:
It ensures that API calls to GitHub are authenticated. The CircleCI IP addresses make enough unauthenticated API calls that they hit the rate limit. Best not to check it in as I think it is tied to my GitHub account.

Does CircleCI define an environment variable that we can use to make this line conditional?